<!--StartFragment-->


## **Report on p53 Protein Modeling and Visualization**

Authors (@slack): Chamss Daghsni (@ChamssDaghsni), Alle Manideep (@Manideep) 


### **Introduction**

The p53 is a tumor suppressor protein which plays a pivotal role in cancer research. It controls the cell cycle and prevents tumor formation. P53 gene's mutations are associated with more than 50% of human cancers, proving its significance as a target for therapeutic intervention.


### **Protein Structure Download**

We downloaded the crystal structure of p53 (ID: 1TUP) from Protein Data Bank.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXdx9tS1RH36q80r8fRGpphqiaalf37t5CvH-TmqvkIfhp6nFuHM0wwGn5o_YhNQI6kfzMaOZli1rbT9mcKHw3Gh3TRkrAoh5SFPx_v5ijultaIBG6Qw00qQzVvDxmOTAOUQL9jgTRpm2WZzzGriqAoeiyQO?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 1:_** _Search Results for p53 Protein (PDB ID: 1TUP) from the Protein Data Bank_




### **Homology Modeling**

Using the provided crystal structure template, SWISS-MODEL generated an homology model of the p53 at a sequence identity of 93.00%.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXdpnWB4S8nw2Sy6V6s32m-L8EmvZX8w3sye3DJx4AapAACEaoOhTQtoW6ANYqdT9cHwQ4nfdRMLYMdtpjWK2N0KnaaSoi06ZKm-aTXoi8E3TJaZfySUGMD3dyJ5gwX3Hk39dAkotrcQlgq4UPrycdSDrayw?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 2:_** _Homology Model of p53 Generated by SWISS-MODEL_


### **AlphaFold Modeling**

We used AlphaFold to predict the structure of p53 and obtained a highly accurate prediction model, with a predicted local distance difference test (pLDDT) of 90. 

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXeP7dLA_KAC5Ta-JcGOBOYxBls0S0oZZwFEXwzlS4LUypca5NLbpGQcUperifXCXScUmKFHooHKYod-off3NTUBwIYx505QOsB-zZFuY2D7B75Mkxfh9AXhsUo-U755tFB4p2PPAW2xVIJCb1-eLKBkYGxJ?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 3:_** _AlphaFold Prediction of p53 Structure_


### **Protein Structure Visualization and Analysis**

Using PyMOL, we visualized the DNA binding domain of p53, its homology model, and its AlphaFold prediction. The protein consists of a DNA-binding domain, a tetramerization domain, and a transactivation domain. Key mutations, like R248 and R273, distort its DNA binding and tumor-suppressing activity. The crystal structure reveals a common immunoglobulin-like β-sandwich fold, generally conserved with RMSD values less than 1 Å. The AlphaFold model more closely aligns with the crystal structure than the homology model, especially around the loops.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXfpPGO754_Q5LF86OLCbsXB3eSDMIM2lu9lGsXdU3wXGkGYh59SArUt83edI0iW6VLz38cO9NPtoeufuqDyYcpZUp9p9mv2gHMnhAKip9LaUyoQ0vZRMC55O5vNEXP6jIlLtCl4yhoE21CJx-LyprAqXjjL?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 4:_** _Visualization of p53 Structures in PyMOL_


### **Modeling Techniques Evaluation**

Homology modeling and AlphaFold successfully predicted p53's structure, with homology modeling being only accurate with available templates but computationally efficient. However, AlphaFold uses deep learning methods for template-free structure prediction with precised details in the loop regions.

After structural alignment and RMSD measurements, the homology model showed an RMSD value of 1.8 Å while the AlphaFold model 2.5 Å. The crystal structure represents the Apo conformation of p53 due to the unbound DNA, suggesting that both models likely assume the Apo conformation.

Comparing these models to the Agonist conformation of p53 bound to DNA, the homology and AlphaFold model, respectively, had an RMSD of 2.2 Å and 3.0 Å, resulting in the homology model capturing the active DNA-bound state well.

The AlphaFold correctly predicted the sites of inhibitor binding for the Antagonist conformation, where p53 is bound to an inhibitor, at an RMSD of 3.8 Å while the homology model at 4.0 Å.




### **Conclusion**

In conclusion, we believe that AlphaFold is the more useful technique for modeling the p53, since it could generate highly accurate predictions without templates. It's an accurate tool in the investigation of structures from cancer-associated proteins and identification of potential drug targets.



<!--EndFragment-->
