<!--StartFragment-->

# Report on p53 Protein Modeling and Visualization

Authors (@slack): Chamss Daghsni (@ChamssDaghsni), Alle Manideep (@Manideep) 


## **Introduction**

<!--StartFragment-->

The p53 is a tumor suppressor protein which plays a pivotal role in cancer research. It controls the cell cycle and prevents tumor formation. P53 gene's mutations are associated with more than 50% of human cancers, proving its significance as a target for therapeutic intervention.


## **Protein Structure Download**

We downloaded the crystal structure of p53 (ID: 1TUP) from Protein Data Bank.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXdx9tS1RH36q80r8fRGpphqiaalf37t5CvH-TmqvkIfhp6nFuHM0wwGn5o_YhNQI6kfzMaOZli1rbT9mcKHw3Gh3TRkrAoh5SFPx_v5ijultaIBG6Qw00qQzVvDxmOTAOUQL9jgTRpm2WZzzGriqAoeiyQO?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 1:_** _Search Results for p53 Protein (PDB ID: 1TUP) from the Protein Data Bank_


## **Homology Modeling**

Using the provided crystal structure template, SWISS-MODEL generated a homology model of the p53 protein at a sequence identity of 93.00%.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXdpnWB4S8nw2Sy6V6s32m-L8EmvZX8w3sye3DJx4AapAACEaoOhTQtoW6ANYqdT9cHwQ4nfdRMLYMdtpjWK2N0KnaaSoi06ZKm-aTXoi8E3TJaZfySUGMD3dyJ5gwX3Hk39dAkotrcQlgq4UPrycdSDrayw?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 2:_** _Homology Model of p53 Generated by SWISS-MODEL_


## **AlphaFold Modeling**

We used AlphaFold to predict the structure of p53 from its amino acid sequence and obtained a high-confidence model, with a predicted local distance difference test (pLDDT) of 90, refelecting a highly accurate prediction.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXeP7dLA_KAC5Ta-JcGOBOYxBls0S0oZZwFEXwzlS4LUypca5NLbpGQcUperifXCXScUmKFHooHKYod-off3NTUBwIYx505QOsB-zZFuY2D7B75Mkxfh9AXhsUo-U755tFB4p2PPAW2xVIJCb1-eLKBkYGxJ?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 3:_** _AlphaFold Prediction of p53 Structure_


## **Protein Structure Visualization and Analysis**

Using PyMOL, we visualized the crystal structure of the DNA binding domain of p53, its homology model, and its AlphaFold prediction. In general, the p53 protein contains an overall domain structure: a DNA-binding domain, a tetramerization domain, and a transactivation domain. Key mutations, like R248 and R273, distort its DNA binding and, therefore, its tumor-suppressing activity. Structures reveal the common immunoglobulin-like β-sandwich fold: a central β-sheet surrounded by loops and helices.

Structural comparisons show that the fold is generally conserved; all Cα atoms have RMSD values less than 1 Å. The AlphaFold model more closely aligns with the crystal structure than the homology model, especially around the loops.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXfpPGO754_Q5LF86OLCbsXB3eSDMIM2lu9lGsXdU3wXGkGYh59SArUt83edI0iW6VLz38cO9NPtoeufuqDyYcpZUp9p9mv2gHMnhAKip9LaUyoQ0vZRMC55O5vNEXP6jIlLtCl4yhoE21CJx-LyprAqXjjL?key=eTpjbmFukqTa7OqFgqCDZQ)

**_Figure 4:_** _Visualization of p53 Structures in PyMOL_


## **Modeling Techniques Evaluation**

Both homology modeling and AlphaFold predict the structure of p53 effectively. Homology modeling is accurate only to the extent of available template structures but is computationally efficient. However, AlphaFold makes template-free structure prediction using deep learning and is suitable for cases involving novel folds of proteins. Thus, AlphaFold seems to capture the details of the loop regions better.

After structural alignment and RMSD measurements, the homology model showed an RMSD value of 1.8 Å while the AlphaFold model had an RMSD of 2.5 Å. The crystal structure represents the Apo conformation of p53 due to the unbound DNA. Thus, both models would more likely assume the unbound (Apo) state of p53.


## **Conclusion**

In conclusion, we believe that AlphaFold is the more useful technique for modeling the p53 protein, since it could generate highly accurate predictions without templates. This makes it a very accurate tool in the investigation of structures from cancer-associated proteins and identification of potential drug targets.

<!--EndFragment-->

